source/documentation/deploying-an-app/use-circleci-to-upgrade-app.html.md.erb

```diff
@@ -71,62 +161,98 @@ To authenticate with ECR, you will need to set:
 #### Kubernetes `ServiceAccount` credentials
 Since a single CircleCI project will need to access multiple namespaces in kubernetes (the environments), it will also need to handle multiple credentials. To simplify authentication, we provide a helper script in our supported [build image](https://github.com/ministryofjustice/cloud-platform-tools-image). For a usage example, see [Deploy To Kubernetes](/documentation/deploying-an-app/use-circleci-to-upgrade-app.html#upload-to-ecr) below.
 
-There are four different variables that CircleCI will need to access *per environment*. Our helper script expects environment variables to be named according to the list below where `<ENVIRONMENT>` should be replaced by some identifier of your choosing (eg.: `STAGING`, `PRODUCTION`).
+There are four different variables that CircleCI will need to access the cluster: 
 
-- `KUBE_ENV_<ENVIRONMENT>_NAME` - the full name of the cluster (eg.: `live-1.cloud-platform.service.justice.gov.uk`)
-- `KUBE_ENV_<ENVIRONMENT>_NAMESPACE` - the name of the `Namespace` (see [Create a namespace][env-create])
-- `KUBE_ENV_<ENVIRONMENT>_CACERT` - the CA Certificate for the cluster, can be acquired from the `Secret` that is generated for the `ServiceAccount`
-- `KUBE_ENV_<ENVIRONMENT>_TOKEN` - the access token generated for the `ServiceAccount`. Please note, you should first base64 decode the token value you retrieve from the secret [in the previous section](/documentation/deploying-an-app/use-circleci-to-upgrade-app.html#creating-a-service-account-for-circleci), e.g. `echo <thereallylongstringthatyougetback> | base64 --decode`.
+- `K8S_CLUSTER_NAME` - the full name of the cluster (eg.: `live-1.cloud-platform.service.justice.gov.uk`)
+- `K8S_NAMESPACE` - the name of the `Namespace` (see [Create a namespace][env-create])
+- `K8S_CLUSTER_CERT` - the CA Certificate for the cluster, can be acquired from the `Secret` that is generated for the `ServiceAccount`
+- `K8S_TOKEN` - the access token generated for the `ServiceAccount`. Please note, you should first base64 decode the token value you retrieve from the secret [in the previous section](/documentation/deploying-an-app/use-circleci-to-upgrade-app.html#creating-a-service-account-for-circleci), e.g. `echo <thereallylongstringthatyougetback> | base64 --decode`.
```

Comment:
I'd suggest creating a sub-heading in the "create a service account" section, called something like "Retrieving the serviceaccount credentials", and then linking to that.

Also, it's not actually the "previous" section (i.e. the section immediately before this one). I know that's being picky, but it might confuse some people, especially if their English isn't so great.